samples/bluetooth/broadcast_audio_source/Kconfig

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Is this the correct copyright?

Reply:
I got this from the Kconfig file in `broadcast_audio_sink` sample. How do I check whether this is the correct copyright?

Reply:
If you copied the file that had the copyright, it's probably alright, although since all content have been replaced, it may be better to add your (Demant) as the copyright company, i.e. use `Copyright (c) 2023 Demant A/S` instead